package.json

```diff
@@ -33,27 +33,27 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "debug": "^4.0.1",
-    "electron-installer-common": "^0.5.0",
-    "fs-extra": "^7.0.0",
-    "get-folder-size": "^2.0.0",
+    "debug": "^4.1.1",
+    "electron-installer-common": "^0.6.0",
+    "fs-extra": "^7.0.1",
+    "get-folder-size": "^2.0.1",
     "lodash": "^4.17.4",
-    "pify": "^4.0.0",
-    "semver": "^5.4.1",
+    "pify": "^4.0.1",
+    "semver": "^5.6.0",
     "word-wrap": "^1.2.3",
-    "yargs": "^12.0.2"
+    "yargs": "^12.0.5"
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "eslint": "^5.6.1",
+    "eslint": "^5.12.1",
     "eslint-config-standard": "^12.0.0",
     "eslint-plugin-import": "^2.7.0",
```

Comment:
It doesn't matter too much because of how the CI tests work and how NPM installs things. I'm mostly trying not to cause a bunch of churn in `package.json` unless I have to.

Reply:
Yes, it's not needed, but given that we were updating all other dependencies, I thought I pointed out. No need to make a new commit just for this if you don't want to